### ISSUE TITLE
Auto Size & Place for Company Logo on Report

### ISSUE THREAD

**Issue by JulianTillmann** [Microsoft/ALAppExtensions#592]
Almost all Reports and Documents have the Company Logo in the upper Right Corner. Most companies don't want to custom layout their documents just because they have a different size logo or move the Picture box around. 

In the past we solved this with clausl's solution: using the System.Drawing.Bitmap DotNet to find out the size of a Picture and pad the Image in RDLC layout accordingly. But since it's "Don'tNET" now, we Need another solution. 

Best solution would be to say to the Image box "Fit proportionally" and "Align = Right". If RDLC won't support this, we Need another solution. We're open to your suggestions :) 

If all else false, you can give us Access to System.Drawing.Bitmap and we'll do the rest.

**Comment by IhorHandziuk** [comment]
Hello @JulianTillmann, thank you for your request and we are really sorry for such a late reply. 
 
The codeunit 5080 (Image Handler Management) have some functions for resizing the image. Else, we can add some functions to System.Drawing.Bitmap wrapper that we already have in he base app. Please, let us know if it works for you

**Comment by JulianTillmann** [comment]
Hi @IhorHandziuk thanks for the update. Codeunit 5080 seems to be doing the trick in our case 

Is there anyway to Keep track when you're creating such cool new codeunits? It's hard to follow all the different documentation, GitHubs, etc.

**Comment by IhorHandziuk** [comment]
Unfortunately, there is no definite way that I'm aware of.  The way that I found this particular codeunit was by searching for "resize" and "image" words in the base app.
We are working on improving our documentation and on ways we publish them.
